src/scene.py

```diff
@@ -49,24 +45,23 @@ def _generate_scene(self):
         For now it'll generate scene layout
         Subject to be changed
         """
-        STEPS = 20
-        pos = Point(0, 0)
-        points = [pos]
-        for _ in range(STEPS):
-            pos += Point(random.choice([-1, 0, 1]), random.choice([-1, 0, 1]))
-            points.append(pos)
-            points.append(pos + Point(-1, 0))
-            points.append(pos + Point(1, 0))
-            points.append(pos + Point(0, -1))
-            points.append(pos + Point(0, 1))
-
-        points = list(dict.fromkeys(points))  # remove duplicates
-
-        for point in points:
-            Floor(self, point)
 
-        Portal(self, random.choice(points))
-        TrainingDummy(self, Point(1, 0))
+        with open(PATH_TO_CONFIG) as json_file:
```

Comment:
could we move it to separate file and class?
so that we can could call here 
```python
BMPMap.load(self, "./src/maps/map1.bmp")
```
or something like that

Reply:
Yes please!